.github/ISSUE_TEMPLATE/ar_bug_report.md

```diff
@@ -1,5 +1,5 @@
 ---
-name: ar_bug_report.md
+name: Automated_Review_bug_template.md
```

Comment:
suggestion: Could this be `Automated Review bug template` instead (no underscores or `.md` extension)

Reply:
Good catch, Tom.